apps/storybook/src/NotificationMarker.stories.tsx

```diff
@@ -0,0 +1,147 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  Text,
+  NotificationMarker,
+  NotificationMarkerProps,
+  Button,
+  IconButton,
+} from '@itwin/itwinui-react';
+import { SvgNotification } from '@itwin/itwinui-icons-react';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  component: NotificationMarker,
+  args: {
+    enabled: true,
+    pulsing: false,
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    status: {
+      control: 'radio',
+      options: ['primary', 'positive', 'warning', 'negative', 'white'],
+    },
+  },
+  title: 'Core/NotificationMarker',
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          display: 'grid',
+          placeItems: 'start',
+          gap: '20px',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta<NotificationMarkerProps>;
+
+export const Basic: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <NotificationMarker {...args}>
+        <SvgNotification width={16} height={16} fill='currentColor' />
+      </NotificationMarker>
+      <NotificationMarker {...args}>Normal</NotificationMarker>
+    </>
+  );
+};
+
+Basic.args = {
+  status: 'primary',
+};
+
+export const WithButtons: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <IconButton styleType='borderless' onClick={action('clicked')}>
```

Comment:
no need for click action imo. we want to show the notification marker, so the less code in the surrounding elements, the better

Reply:
Removed